lib/ansible/modules/cloud/azure/azure_rm_manageddisk.py

```diff
@@ -93,6 +93,17 @@
             - To detach a disk from a vm, explicitly set to ''.
             - If this option is unset, the value will not be changed.
         version_added: 2.5
+    attach_caching:
+        description:
+            - "Disk caching policy controlled by VM. Will be used when attached to the VM defined by C(managed_by)"
+            - "If this option is different from the current caching policy,"
+            - "the managed disk will be deattached from the VM and attached with current caching option again"
```

Comment:
The long line can be written as the snippet. The second line should start as the same indent as the previous one. Here if we write into two list items(start from `-` or `*`), the documentation will be in two sentences.

```
            - "If this option is different from the current caching policy,
               the managed disk will be deattached from the VM and attached with current caching option again"
```

Reply:
Thanks! I fix the doc change. Besides, I check some documentation parts in other files and delete the double quotes.

Reply:
If one sentence is written in two lines, the quotes are required. 😃

Reply:
😮 oh. I fix it again. So, if the sentence is written in one line, are the quotes optional?

Reply:
yes, the documentation should be a valid yaml anyway. You can find an online website to check the linting for documentation part.